libs/block-content/website/src/lib/teaser/teaser.tsx

```diff
@@ -196,32 +289,52 @@ export const Teaser = ({teaser, alignment, className}: BuilderTeaserProps) => {
 
   const {
     date,
-    elements: {H5, H6, Link, Image}
+    elements: {Link, Image}
   } = useWebsiteBuilder()
 
-  return (
-    <TeaserWrapper {...alignment}>
-      <Link color="inherit" underline="none" href={href} css={teaserLinkStyles}>
-        <TeaserInnerWrapper className={className}>
-          {image && <Image image={image} css={teaserImageStyles} />}
+  const linkStyles = useMemo(() => teaserLinkStyles(theme), [theme])
+  const enhancedImageStyles = useMemo(
+    () => getEnhancedImageStyles(theme, isHovered),
+    [theme, isHovered]
+  )
 
-          <TeaserContent>
+  return (
+    <TeaserWrapper
+      {...alignment}
+      onMouseEnter={() => setIsHovered(true)}
```

Comment:
This has many accessibility issues and can bug out in multiple ways. You can use react-aria for this or css.

Reply:
I tried using css but due to the structure, using of website-builder components it was impossible to implement.

Reply:
How so?

Reply:
ok I used react-aria